content/blog/eclipse-temurin-slsa2/index.md

```diff
@@ -0,0 +1,110 @@
+  SBOM.  If you want to join in the discussion on the content, you can find
+  the conversation in
+  [temurin-build#3013](https://github.com/adoptium/temurin-build/issues/3013)
+  or talk to us in the #secure-dev slack channel in the
+  [Adoptium workspace](https://adoptium.net/slack/).
+
+## SLSA Level 2
+
+Level 2 adds in additional requirements to provide some tamper resistance of
+the build process, including having all of our code version controlled.  We
+achieve this through our use of GitHub for both the OpenJDK product code and
+the code that runs the build and distribution processes through our Jenkins
+CI server.  The requirements here are:
+
+- **Source - Version controlled**. All of our source code is stored in GitHub in
+  the jdkXX repositories such as
+  [jdk17u](https://github.com/adoptium/jdk17u/), which we mirror from the
+  [openjdk project](https://github.com/openjdk). These
+  are version controlled and have tags for each release which we build from.
+  Contributors agree to the rules of the
+  [Eclipse Contributor Agreement (ECA)](https://www.eclipse.org/legal/ECA.php)
+
+- **Build - Build service**. All of the build steps are run using our Jenkins
+  build service (https://ci.adoptopenjdk.net) which is used to
+  perform the builds, generate the SBOMs, and build the installers where
+  applicable. The output from the builds are then posted into GitHub release
+  repositories named as temurinXX-binaries (e.g.
+  [temurin17-binaries](https://github.com/adoptium/temurin17-binaries) and
+  also exposed via our API and web site.
+
+- **Provenance - Authenticated**. We sign the SBOMs for the latest set of
+  releases to guarantee their authenticity.
+
+- **Provenance - service generated**. The GPG signatures for the binary
+  are generated as an integral part of the build process for the binaries
+  and will be done in the same way for the signing of the SBOMs - the current
+  SBOM signing was done retrospectively in our Jenkins instance. The Jenkins
+  instance hosts the private keys from the Eclipse Foundation which are used
+  to perform the signature generation.
+
+# Next steps for the project
+
+Are we finished? Absolutely not! We are continuing to work towards
+achieving the higher levels of SLSA and have already achieved many of the
+requirements of higher levels. But for now, we are proud to be able to claim
+full compliance with SLSA level 2.
+
+We are currently tracking forward progress to meeting the higher level
+requirements at
+[adoptium#160](https://github.com/adoptium/adoptium/issues/160). We already
+meet some of the requirements of higher levels - in some cases there are
+differences in the criteria which we meet on different platforms. For
+example, we have worked to [make our build process reproducible](https://blog.adoptium.net/2022/06/adoptium-reproducible-builds/)
+and on Linux, Windows and MacOS for JDK19+ we are able to produce binary
```

Comment:
Can it now be claimed as 17+ ?

Reply:
I wasn't clear that everythign has been backported on all platforms yet so I'll defer to @andrew-m-leonard on that.  It can always be updated later on as things move forward ;-)

Reply:
jdk17 yes, all backports from 19 to 17 that are needed are done. but of course not jdk18, so jdk17+ is not quite accurate, it should be:
**jdk17, and jdk19+**

Reply:
@andrew-m-leonard Does that mean that the latest 17.0.5+8 release was reproducible on all three operating systems?

Reply:
i'm going to draw up a "table" which should make it clearer, also got plans on possible nightly build-repo...

Reply:
reworded to `on Linux, Windows and MacOS for JDK17 (LTS) and JDK19+ we have created
and shipped binary reproducible builds`